src/redux/reducers/things.ts

```diff
@@ -1,33 +1,39 @@
-import actions from "../actions";
-import { ThingType } from "../../models/thing";
+import actions from '../actions';
+import {ThingType} from '../../models/thing';
 
 const things = (state: ThingType[] = [], action: any) => {
-    const clone = [...state];
-
-    switch (action.type) {
-        case actions.ADD_THING:
-            clone.push(action.thing);
-
-            return clone;
-        case actions.EDIT_THING:
-            const selectedThingIndex = clone.findIndex(t => t.id === action.thing.id);
-
-            clone.splice(selectedThingIndex, 1, action.thing);
-
-            return clone;
-        case actions.DELETE_THING:
-            const selectedThingIndexTBD = clone.findIndex(t => t.id === action.id);
-            clone.splice(selectedThingIndexTBD, 1);
-
-            return clone;
-        case actions.COMPLETE_THING:
-
-            // DO STUFF
-
-            return clone;
-        default:
-            return state;
-    }
+  const clone = [...state];
+
+  switch (action.type) {
+    case actions.ADD_THING:
+      clone.push(action.thing);
+
+      return clone;
+    case actions.EDIT_THING:
+      const selectedThingIndex = clone.findIndex(
+        (t) => t.id === action.thing.id,
+      );
+
+      clone.splice(selectedThingIndex, 1, action.thing);
+
+      return clone;
+    case actions.DELETE_THING:
+      const selectedThingIndexTBD = clone.findIndex((t) => t.id === action.id);
+      clone.splice(selectedThingIndexTBD, 1);
+
+      return clone;
+    case actions.SESSION_THING:
```

Comment:
create its own file in src/redux/reducers/sessions.ts with all actions for the session